docs/conf.py

```diff
@@ -40,6 +52,32 @@
 
 language = None
 
+nbsphinx_thumbnails = {
+    'notebooks/usecase_emg_sign_classification_in_spd_manifold':'../../notebooks/figures/paper_rock_scissors.png',
+    'notebooks/usecase_graph_embedding_and_clustering_in_hyperbolic_space':'../../notebooks/figures/karate_graph.png',
```

Comment:
`DeepSource` (the software we use to check for correct coding style) complains about the lack of a space after the `:` and at other locations in this file (see [here](https://deepsource.io/gh/geomstats/geomstats/run/9f4a1f87-54b9-4902-b4ac-af93ef665349/python/)). Could you correct the code style? Thank you 🙏